### DISCUSSION TITLE
marseille blitz pgn after loading custom pgn

### DISCUSSION THREAD

**Discussion by arnaule** [rodpolako/Chess-PGN-Trainer#6] [General]
Congratulation for this excellent app!
When I load my pgn, it displays puzzle 1/2 and the 2nd (which was not manually loaded) is Marseille Blitz
Do you know how to disable this ?
Thank you,
Arnaud

**Comment by rodpolako** [comment]
Can you attach the PGN?  I'm not exactly sure I understand the question.

If you only want to see a single puzzle, remove all others and save the PGN and that should do it.

**Comment by arnaule** [comment]
Yes thank you that was it, it was indeed in my PGN!
